e2e-tests/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
As this is an e2e of the whole stack, it makes some sense to put this in the root `package.json`

Reply:
yeah, i'm kinda open for discussion on this.. we could just bin the e2e-tests folder structure and have cypress hanging right off the root, which would move all the relevant cypress config files into folder root..

I don't have many rules in life, but I try to stick to no more than 7 files in a folder. 

You have to draw the line -somewhere- and that's as a good an arbitrary number as any.. basically it means you cant just bang everything in the root because you end up with a massive list of .rcthis + .editorthat etc. etc. which are sometimes good for all modules but not always.. so I more view the existence of that e2e-tests folder as "a way to stop accumulating as much in project root"..

but it's not my project, so... my random sensibilities may not apply... either way we'll re-visit when i'm proposing a merge + it's not complex to move.